logstash-core-event-java/build.gradle

```diff
@@ -0,0 +1,104 @@
+buildscript {
+  repositories {
+    mavenLocal()
+    mavenCentral()
+    jcenter()
+  }
+  dependencies {
+    classpath 'net.saliman:gradle-cobertura-plugin:2.2.8'
+    classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.2'
+  }
+}
+
+//allprojects {
```

Comment:
yeah, so I kept them around just to remember the syntax. eventually we'll definitely want to migrate our base logstash rake stuff to gradle and in that context it will in fact control subprojects. but in a any case, we'll manage them, I will cleanup here.